src/index.html

```diff
@@ -20,8 +20,380 @@
       href="styles/index.scss"
```

Comment:
You are linking a SCSS file in your HTML. Browsers can't interpret SCSS, they can only interpret CSS. You need to compile your SCSS to CSS and link the CSS file here. There should be a task in your build process that does this.